frontend/src/components/Topbar.jsx

```
@@ -69,9 +69,25 @@ export const Topbar = () => {
   return (
     <>
       <div className={styles.container}>
-        <div className={styles.logo} onClick={() => navigate('/dashboard')}>STUDIFY</div>
+        <div
+          className={styles.logo}
+          role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)